setup.py

```diff
@@ -97,6 +97,8 @@
     install_requires=[
         "numpy",
         "tokenizers == 0.5.2",
+        # dataclasses for Python versions that don't have it
+        "dataclasses;python_version<'3.7'",
```

Comment:
https://stackoverflow.com/a/32643122/593036

Reply:
(This is not a backport or a "polyfill" of `dataclasses` but the actual original implementation – that was then integrated into Python from 3.7)